server/jamovi/server/instance.py

```diff
@@ -461,6 +461,8 @@ def _on_part_saved(self, request, result):
     def _on_open(self, request):
         path = request.filename
 
+        if path != '':
+            self._data.suspend_data_tracking()
```

Comment:
should this be inside `formatio.read()`?

Reply:
I have not put it in there because it needs to be outside the try catch

Reply:
actually, does it even matter? (if `self._data.resume_data_tracking()` is called or not when an exception is thrown during import?) ... if an exception is thrown, the data set is discarded anyway?

Reply:
yeah, because the blank sheet still wants tracking enabled

Reply:
So i took the stance that tracking is always on, except on opening. So if it is turned off, and then cancelled, it needs turning back on.

Reply:
but the blank sheet isn't going to raise an exception ...